src/main/java/org/rutebanken/tiamat/rest/graphql/fetchers/ParkingUpdater.java

```diff
@@ -332,4 +351,22 @@ private ParkingArea resolveSingleParkingArea(Map input) {
         area.setParkingProperties(resolveSingleParkingProperties((Map) input.get(PARKING_PROPERTIES)));
         return area;
     }
+
+    private AccessibilityAssessment resolveAccessibilityAssessment(AccessibilityLimitation limitationFromInput) {
+        AccessibilityAssessment accessibilityAssessment =  new AccessibilityAssessment();
```

Comment:
Can you simplify this method e.g. 

private AccessibilityAssessment resolveAccessibilityAssessment(AccessibilityLimitation limitation) {
AccessibilityAssessment assessment = new AccessibilityAssessment();
assessment.setLimitations(List.of(limitation));
return assessment;